### ISSUE TITLE
Is gen_funcidx interface required?

### ISSUE THREAD

**Issue by keceli** [wavefunction91/ExchCXX#26]
I have a problem with `cppyy` generated bindings rooted down to `gen_funcidx`:
```
 File "/app/NUX/nux/nux.py", line 15, in <module>
    from nwchemex import chemist, TA, chemcache, simde, nwchemex, pluginplay, scf, parallelzone
  File "/app/NUX/build/Python/nwchemex/__init__.py", line 13, in <module>
    cppyy.add_include_path(p)
  File "/usr/local/lib/python3.10/dist-packages/cppyy/__init__.py", line 264, in add_include_path
    raise OSError('No such directory: %s' % path)
OSError: No such directory: /app/NUX/build/_deps/libxc-build/gen_funcidx
```
When I build ExchCXX, I don't see this directory being created. I built and run tests after removing this [line](
https://github.com/wavefunction91/ExchCXX/blob/0b2119e00d0b3117bb943f83fcef9a047d61bde4/CMakeLists.txt#L87) from `CMakeLists.txt` and all was fine, so I am curious if this line is required, or not.

**Comment by wavefunction91** [comment]
PR the change and see if it passes CI - this was an *ad hoc* solution back when Libxc didn't have a proper CMake build system.

**Comment by keceli** [comment]
PR #28  passes CI.

**Comment by wavefunction91** [comment]
Closed by #28
